src/Libraries/Microsoft.Extensions.AI.Evaluation.NLP/BLEUEvaluatorContext.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma warning disable S3604
+// S3604: Member initializer values should not be redundant.
+// We disable this warning because it is a false positive arising from the analyzer's lack of support for C#'s primary
+// constructor syntax.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Extensions.AI.Evaluation.NLP;
+
+/// <summary>
+/// Contextual information that the <see cref="BLEUEvaluator"/> uses to compute the BLEU score for a response.
+/// </summary>
+/// <remarks>
+/// <see cref="BLEUEvaluator"/> measures the BLEU score of a response compared to a reference. BLEU (Bilingual Evaluation Understudy)
+/// is a metric used to evaluate the quality if machine-generated text.
+/// </remarks>
+public sealed class BLEUEvaluatorContext : EvaluationContext
+{
+    /// <summary>
+    /// Gets the unique <see cref="EvaluationContext.Name"/> that is used for
+    /// <see cref="BLEUEvaluatorContext"/>.
+    /// </summary>
+    public static string BLEUContext => "BLEU Context";
```

Comment:
I know its longer - but I'd prefer the following name so that we match the naming convention we followed in all other contexts :)

```suggestion
    public static string BLEUEvaluatorContextName => "BLEU Context";
```